lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/ClassHierarchyImpl.cs

```diff
@@ -287,7 +287,7 @@ public INode BuildPathToNode(Type type)
             {
                 IDictionary<string, string> mapping = null;
                 _aliasLookupTable.TryGetValue(np.GetAliasLanguage().ToString(), out mapping);
-                if (null == mapping)
+                if (mapping == null)
```

Comment:
A good chance to rewrite it to more idiomatic
```C#
var aliasLanguage = np.GetAliasLanguage().ToString();
IDictionary<string, string> mapping;
if (!_aliasLookupTable.TryGetValue(aliasLanguage, out mapping))
{
    mapping = new Dictionary<string, string>();
    _aliasLookupTable.Add(aliasLanguage, mapping);
}
```

Reply:
Good idea. Will do.